web/src/components/forms/JiraDestinationForm/JiraDestinationForm.test.tsx

```diff
@@ -68,6 +71,7 @@ describe('JiraDestinationForm', () => {
     const apiKeyField = getByLabelText('* Jira API Key');
     const assigneeIdField = getByLabelText('Assignee ID');
     const issueTypeField = getByLabelText('* Issue Type');
+    const labelsField = getByLabelText('Labels', { selector: 'input' });
```

Comment:
Optional: In most tests, we write this as:

```suggestion
    const labelsField = getByAllLabelText('Labels')[0];
```

If this change is implemented, it should affect all similar statements in this page